### PR TITLE
Run asv in test suite.

### PR DESCRIPTION
**Pull request by bjodah**
This will give PR authors feedback on whether `asv run` works on their branch.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,9 @@ python:
   - "2.7"
 install:
   - pip install -r requirements.txt
-  - pip install pytest
```
Comment (debugger22) [inline .travis.yml]: Why did you remove `pytest`?

Comment (debugger22) [inline .travis.yml]: +1 otherwise.

**Comment by bjodah** [comment]
I removed `pytest` since it's already provided by the travis environment. Tests pass so I am merging this. Thank you for your review.
